bootstrap-salt.sh

```diff
@@ -6576,7 +6576,7 @@ if [ "$_CUSTOM_MASTER_CONFIG" != "null" ] || [ "$_CUSTOM_MINION_CONFIG" != "null
         _TEMP_CONFIG_DIR="$_SALT_ETC_DIR"
     fi
 
-    if [ "$_CONFIG_ONLY" -eq $BS_TRUE ]; then
+    if [ ${NO_DEPS} -eq $BS_FALSE -a "$_CONFIG_ONLY" -eq $BS_TRUE ]; then
```

Comment:
`${NO_DEPS}` should be `${_NO_DEPS}` here, yes?

Reply:
yep, my bad. you're right. the variable should've been _NO_DEPS.